S100SapCallback_OpenApi_Spec.yml

```diff
@@ -0,0 +1,151 @@
+openapi: 3.0.1
+info:
+  title: UKHO S100 ERP Facade Service API
+  description: |
+    The UKHO S100 ERP Facade Service API handles callback event received from SAP.
+  contact:
+    name: UKHO
+    email: contact@ukho.co.uk
+  version: 0.0.1
+servers:
+  - url: https://erpfacade-dev-api.azurewebsites.net
+    description: Dev URL
+  
+paths:
+  /s100erpfacade/callback/{correlationId}:
+    get:
```

Comment:
Following REST concepts, this isn't a GET, but should really be a POST.

Reply:
Updated to POST request. 
Thanks